### PR TITLE
London Class 7 - Oksana Kostenko - JavaScript-Core-3-Coursework-Week3

### PR DESCRIPTION
**Pull request by oksygenn**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name: Oksana Kostenko
- Your City: London
- Your Slack Name: Oksana Kostenko

# Homework Details

- Module: JS Core3
- Week: 3

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

### CODE REVIEW

File: exercises/exercise-2/exercise-2.js [hunk 1]
```diff
@@ -1,12 +1,86 @@
...
+    occupation: "Teacher",
+  },
+];
+
+// TASK 1
+for (const { firstName, lastName, house } of hogwarts) {
+  if (house === "Gryffindor") {
+    console.log(firstName, lastName);
+  }
+}
```
Comment (ASLANDENIZ) [inline exercises/exercise-2/exercise-2.js:79]: that's a great solution, super short, well done! I really like it. :)

File: exercises/exercise-3/exercise-3.js [hunk 2]
```diff
@@ -1,9 +1,24 @@
...
+
+const createReceipt = (orderObj) => {
+  let toPay = 0;
+
+  console.log(`QTY ITEM TOTAL`);
+
+  for (const { itemName, quantity, unitPrice } of orderObj) {
+    const unitTotal = unitPrice * quantity;
+    toPay += unitTotal;
+    console.log(`${quantity} ${itemName} ${unitTotal.toFixed(2)}`);
```
Comment (ASLANDENIZ) [inline exercises/exercise-3/exercise-3.js:18]: if you want them on the same line you can use \t (tab) for space for all your console logs with a condition if the item name is greater than a certain length,  great work!

**Review by ASLANDENIZ — COMMENTED** [review]
I finished my review, I like how quick and handy are your solutions. Well done!

File: exercises/exercise-3/exercise-3.js (hunk 2 above)
Comment (oksygenn) [reply exercises/exercise-3/exercise-3.js:18]: Hi Deniz! Thanks for the suggestion! I will do that 👍

**Pull request closed without merging**